packages/shared/lib/services/notification/slack.service.ts

```diff
@@ -470,6 +472,7 @@ export class SlackService {
                 type,
                 connection_list: [nangoConnection.id as number]
             })
+            .forUpdate()
```

Comment:
this is a INSERT so `forUpdate` doesn't apply. Not sure why knex allows it

Reply:
Updated